src/main/java/Duke/Command.java

```diff
@@ -0,0 +1,142 @@
+package Duke;
+
+import common.Messages;
+import data.*;
```

Comment:
Imported classes should be listed explicitly (i.e. `import data.Deadline` instead of `import data.*`). Please check for this problem for the rest of your classes as well.